versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.26"
+      version = ">= 5.32"
```

Comment:
we need to change this throughout - aligning all of the examples as well

Reply:
Thanks! I've updated the required version in the `examples` and updated the docs. Let me know if there anything else I've missed :)